app/controllers/hyrax/embargoes_controller.rb

```diff
@@ -1,5 +1,56 @@
 module Hyrax
   class EmbargoesController < ApplicationController
-    include Hyrax::EmbargoesControllerBehavior
+    include Hyrax::Collections::AcceptsBatches
+
+    def index
+      authorize! :index, Hydra::AccessControls::Embargo
+    end
+
+    def edit
+      work = find_resource(params[:id])
+      authorize! :edit, work
+      @resource = find_resource(work.embargo_id)
+      @change_set = EmbargoChangeSet.new(@resource, parent_resource: work)
+    end
+
+    # Removes a single embargo
+    def destroy
+      work = find_resource(params[:id])
+      authorize! :destroy, work
+      embargo = Hyrax::Actors::EmbargoActor.new(work).destroy
```

Comment:
I'd prefer `destroyed_embargo` local variable. My idiomatic assumption for `#destroy` method calls is a Boolean return value.

Reply:
I haven't changed this code, simply moved it from a module to this class.